README.md

```diff
@@ -1100,6 +1100,7 @@ Holds parameters to initialize the SDK.
 | **`shouldShowInAppMessagesAutomatically`**      | <code>boolean</code>                                                                    | Whether we should show store in-app messages automatically. Both Google Play and the App Store provide in-app messages for some situations like billing issues. By default, those messages will be shown automatically. This allows to disable that behavior, so you can display those messages at your convenience. For more information, check: https://rev.cat/storekit-message and https://rev.cat/googleplayinappmessaging                                                           |
 | **`entitlementVerificationMode`**               | <code><a href="#entitlement_verification_mode">ENTITLEMENT_VERIFICATION_MODE</a></code> | Verification strictness levels for [EntitlementInfo]. See https://rev.cat/trusted-entitlements for more info.                                                                                                                                                                                                                                                                                                                                                                             |
 | **`pendingTransactionsForPrepaidPlansEnabled`** | <code>boolean</code>                                                                    | Enable this setting if you want to allow pending purchases for prepaid subscriptions (only supported in Google Play). Note that entitlements are not granted until payment is done. Disabled by default.                                                                                                                                                                                                                                                                                  |
+| **`diagnosticsEnabled`**                        | <code>boolean</code>                                                                    | Enabling diagnostics will send some performance and debugging information from the SDK to our servers. Examples of this information include response times, cache hits or error codes. No personal identifiable information will be collected. The default value is false.                                                                                                                                                                                                                |
```

Comment:
`our` -> `RevenueCat's` maybe to make it clear whose servers we're talking about?

Reply:
Hmm you're right, however, this doc actually comes from PHC, so I will need to make the update there. This should pick it up on the next release then. Will do the change there!

Reply:
Done in https://github.com/RevenueCat/purchases-hybrid-common/pull/1001